docs/source/index.rst

```diff
@@ -36,6 +36,36 @@ user:
     print(kennethreitz.followers_count)
 
     followers = [str(f) for f in gh.followers('kennethreitz')]
+    
+    # create a repository
+    gh.create_repository(name='Repo_Name')
+    
+    # grab the repo
+    repos = list(gh.repositories())
+    my_repo = gh.repository(gh.me(), 'Repo_Name') # grab match
+    
+    # create a label
```

Comment:
Are the `create_*` method names not clear enough as to what is happening that we need these comments?

Reply:
I don't think they hurt in an example. 
Also this groups each of them actions nicely. Like headlines.